readthedocs.yml

```diff
@@ -1,5 +1,4 @@
 conda:
-    file: docs/environment.yml
+    file: docs/conda_env.yml
```

Comment:
I think this should stay as `environment.yml`. That is the standard name for conda environment files.